### PR TITLE
Add shadow-20 and white background to box-like elements

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

- [x] add white background-color to the box-like elements
- [x] add a box shadow to those same elements

## Screenshots

|  Before  |  After  |
|--------|--------|
|Screenshot 2024-08-07 at 4 01 21 PM|Screenshot 2024-08-07 at 4 01 05 PM|

## Testing in `kajabi-products`

1. (**LOW/MEDIUM/HIGH/BREAKING**) Description of the change and its impact with QA as the audience.
   - [ ] One more examples of the component in use to either test the change or verify the change has not had adverse effects.

### CODE REVIEW

**Review by ju-Skinner — APPROVED** [review]

File: packages/sage-assets/lib/stylesheets/layout/_frame.scss [hunk 1]
```diff
@@ -216,6 +216,7 @@ $-sage-frame-wrap: none;
 
 @each $-key, $-val in $-sage-frame-borders {
   .sage-frame--border-#{$-key} {
+    background-color: sage-color(white);
```
Comment (pixelflips) [inline packages/sage-assets/lib/stylesheets/layout/_frame.scss:219]: I may be incorrect here, but is this not related to borders?

Comment (QuintonJason) [reply packages/sage-assets/lib/stylesheets/layout/_frame.scss:219]: The intention is to only add the white background color to any frames that are being used like panels, to serve as only wrappers, without affecting frames that exist solely for layout purposes. Frames without borders, don't need the white background.

**Review by pixelflips — APPROVED** [review]
